engines/pink/gui.cpp

```diff
@@ -146,10 +146,9 @@ struct SaveStateDescriptorTimeComparator {
 	}
 };
 
-static SaveStateList listSaves(bool isPeril) {
+SaveStateList PinkEngine::listSaves() const {
 	Common::SaveFileManager *saveFileMan = g_system->getSavefileManager();
-	Common::String pattern = isPeril ? "peril.s##" : "pokus.s##";
-	Common::StringArray filenames = saveFileMan->listSavefiles(pattern);
+	Common::StringArray filenames = saveFileMan->listSavefiles(_targetName + ".s##");
```

Comment:
In fact, this was incorrect in the first place. The proper way is to use `MetaEngine::getSavegameFile()`.

Reply:
Right, had a feeling that MetaEngine had something for it. I'm switching to the proper way.